### DISCUSSION TITLE
Problems with REOWN useAppKitProvider()

### DISCUSSION THREAD

**Discussion by peermodel** [reown-com/appkit#3127] [Q&A]
The example found at https://docs.reown.com/appkit/javascript/core/installation
has syntax problems:

1.) line "{ walletProvider } = useAppKitProvider()" gives the following error:

Expected 1 arguments, but got 0.ts(2554)
index.d.ts(23, 46): An argument for 'chainNamespace' was not provided.

2.) line "const ethersProvider = new BrowserProvider(walletProvider)" gives the following error:

Argument of type 'unknown' is not assignable to parameter of type 'Eip1193Provider'.

I'm using:

+-- @reown/appkit-adapter-ethers@1.1.6
+-- @reown/appkit@1.1.6
+-- @types/node@22.7.7
+-- @walletconnect/web3-provider@1.8.0
+-- @walletconnect/web3wallet@1.16.1
+-- ethers@6.13.4
+-- typescript@5.6.3
+-- web3@4.13.0
...

Grateful for any advice!

**Comment by gotnoshoeson** [comment]
You need to use an argument for the type of adapter you want. Example for EVM  chain: useAppKitProvider("eip155"). If you start typing a " inside the () of useAppKitProvider, you should get some options show up to choose from.
